src/services/users/index.js

```diff
@@ -123,10 +131,37 @@ const inviteUser = async (email) => {
 	return signupInfo
 }
 
+const getNewTokens = async (email, oldRefreshToken) => {
+	const user = await getUserByEmail(email)
```

Comment:
`getUserByEmail` does more than we want, it will throw an `invalidEmailOrPassword` error for an endpoint that is not concerned with passwords

kindly form a query to get the email here

Reply:
actually on second thought, no need to make a request for the user by email before we check the refresh token

kindly make the query check both the email and refresh Token at the same time. if there's no entry, you throw the custom  error